uptime/checker.go

```diff
@@ -0,0 +1,416 @@
+package uptime
+
+import (
+	"context"
+	"sync"
+	"fmt"
+	"time"
+
+	logging "github.com/ipfs/go-log/v2"
+
+	"github.com/filecoin-project/lotus/api/v0api"
+	"github.com/filecoin-project/go-address"
+	chainTypes "github.com/filecoin-project/lotus/chain/types"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/big"
+
+	"github.com/libp2p/go-libp2p/p2p/protocol/ping"
+	"github.com/libp2p/go-libp2p-core/host"
+	libp2pMultiaddr "github.com/multiformats/go-multiaddr"
+	peerstore "github.com/libp2p/go-libp2p-core/peer"
+)
+
+var log = logging.Logger("uptime")
+
+const NEW_CHECKER = 2
```

Comment:
Can you add the `METHOD` suffix so we add additional context about what this is.
```suggestion
const NEW_CHECKER_METHOD = 2
```

BTW, Golang convention is to use Camel Case for constants, so strictly speaking it should be `NewChecker`.